sdcm/utils/common.py

```diff
@@ -87,7 +87,7 @@
 LOGGER = logging.getLogger('utils')
 DEFAULT_AWS_REGION = "eu-west-1"
 DOCKER_CGROUP_RE = re.compile("/docker/([0-9a-f]+)")
-SCYLLA_AMI_OWNER_ID = "797456418907"
+SCYLLA_AMI_OWNER_ID = ["797456418907", "158855661827"]
```

Comment:
rename to `SCYLLA_AMI_OWNER_IDS`? or `SCYLLA_AMI_OWNER_ID_LIST`?

Reply:
fixed